code/__HELPERS/type2type.dm

```diff
@@ -358,14 +358,14 @@
 			return list(LEG_RIGHT, FOOT_RIGHT)
 
 //Turns a Body_parts_covered bitfield into a list of organ/limb names.
-//(I challenge you to find a use for this)-I found a use for it!!
-/proc/body_parts_covered2organ_names(bpc)
+//(I challenge you to find a use for this) -I found a use for it!! | So did I!.
+/proc/cover_flags2body_zones(bpc)
 	var/list/covered_parts = list()
 
 	if(!bpc)
 		return 0
 
-	if(bpc & FULL_BODY)
+	if(bpc == FULL_BODY)
```

Comment:
I think this change should be reverted as it will cover more cases.

Reply:
i could, but TG does this way and i'm sure it's supposed to be more precise